### DISCUSSION TITLE
importing DLC pre-labelled files onto SLEAP

### DISCUSSION THREAD

**Discussion by jannhan** [talmolab/sleap#1833] [Help!] [answered]
Can anyone show a screenshot of what is supposed to happen when you import a csv file of already labelled deeplabcut data into SLEAP? Does it make the training faster? What are the steps you still have to do after?

**Comment by talmo** [comment]
Hi @jannhan,

Apologies for the delay! We're a bit behind on support responses at the moment.

You should be able to import training data (manual labels, not predictions) from the File menu.

It should not require any additional steps -- you'll just see the labeled frames imported right away.

This feature allows you to reuse annotations that you've already generated, saving you time from having to start from scratch.

Did you run into issues when using this? If nothing happened in the GUI, did you get any error messages in the terminal?

Cheers,

Talmo

**Comment by jannhan** [comment]
No problem, I totally understand! The first error/warning message that came up said that I had to install libnvinfer.so <http://libnvinfer.so/>.6 and libnvinfe_plugin.so <http://libnvinfe_plugin.so/>.6, and this is after I’ve already downloaded TensorRT, TensorFlow, cuDNN, CUDA, so I’m still stuck. Please let me know if you know what could be going on! 

Thank you so much for your time and help!
Janna

**Reply by talmo** [reply]
Hi @jannhan,

This error can be safely ignored and is just related to GPU dependencies.

I'd advise that you install SLEAP via the [recommended conda package method](https://sleap.ai/installation.html#conda-package), which will install it in an isolated conda environment where it handles GPU dependencies and won't interact with other packages that you have installed.

**Comment by jannhan** [comment]
If I ignore this warning message, and import the DeepLabCut data, an error
comes up that says "ValueError: Skeleton already has node named (bodypart1)"

**Reply by talmo** [reply]
That probably shouldn't be happening -- are you importing it into a blank project or a pre-existing one?

What happens if you try just creating a brand new project and then importing the CSV?

**Comment by jannhan** [comment]
I reinstalled SLEAP with the conda package method and the warning went
away! However, I tried opening a new project then importing the csv and the
same error comes up. What is it supposed to look like when I open it?

**Reply by talmo** [reply]
Hmm, it's supposed to show the imported data, so perhaps something funky is going on.

Do you mind trying out the [v1.4.1a2 prerelease](https://github.com/talmolab/sleap/releases/tag/v1.4.1a2)? I think there might be a fix related to this issue in there.

If it still doesn't work, we'll ask you for your CSV file so we can see what's going on.

**Comment by jannhan** [comment]
Unfortunately the same error still comes up. Here is the CSV file I have
been testing this with

**Reply by talmo** [reply]
Hi @jannhan,

I think you probably tried to attach the file to an email -- I think GitHub doesn't allow this.

Do you mind uploading it to Google Drive or something? You might also be able to attach it from the GitHub website for this discussion --> https://github.com/talmolab/sleap/discussions/1833

Cheers,

Talmo

**Reply by jannhan** [reply]
Sure, let me know if this works
[109_OF_fastedDLC_resnet50_POMC_opto_OFFeb16shuffle1_500000.csv](https://github.com/user-attachments/files/16435439/109_OF_fastedDLC_resnet50_POMC_opto_OFFeb16shuffle1_500000.csv)

**Reply by talmo** [reply, marked as answer]
Hi @jannhan,

Thanks for sharing the CSV! It looks like these are the _predictions_, not the manually labeled annotations.

We could probably add support for importing predictions, but one issue is that this CSV doesn't have paths to the images or the video that the predictions correspond to.

If you have the CSV with the labels that were used to train the model, that should work :)

Cheers,

Talmo

**Reply by jannhan** [reply]
Thank you so so much, the manually labelled csv works!
